tests/test_spidermon_signal_connect.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+import pytest
+from scrapy import signals
+from scrapy.crawler import CrawlerRunner
+from scrapy.spiders import Spider
+
+from spidermon.contrib.scrapy.extensions import Spidermon
+
+
+def get_crawler(settings_dict=None):
+    runner = CrawlerRunner(settings_dict)
+    return runner.create_crawler(Spider)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 4), reason="requires python3.4 or higher")
+def test_spider_opened_connect_signal():
+    from unittest import mock
+
+    with mock.patch.object(Spidermon, "spider_opened") as spider_opened_method:
```

Comment:
I always use `autospec`, since without it you can use imaginable methods and not even know the test doesn't work

Reply:
`autospec` didn't work . Do you have an idea why?